src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -66,7 +68,7 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         try {
             return $this->handleRaw($request, $type);
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
This is a bc break. If I use the HttpKernel standalone, throwable did not trigger the exception event. Now it does.

Reply:
We could change the `$catch` parameter of this method to allow for more fine-grained control over what should be caught.

Reply:
What about just having 2 channels, exception and error, and we hook the `ExceptionListener` on both events. Backward compatible and errors can still be handled in 4.4.